ZenPacks/zenoss/LinuxMonitor/modeler/plugins/zenoss/cmd/linux/df.py

```diff
@@ -1,19 +1,46 @@
 ##############################################################################
 #
-# Copyright (C) Zenoss, Inc. 2016, all rights reserved.
+# Copyright (C) Zenoss, Inc. 2016-2018, all rights reserved.
 #
 # This content is made available according to terms specified in
 # License.zenoss under the directory where your Zenoss product is installed.
 #
 ##############################################################################
+
+"""Model Linux file systems by running "df -PTk"."""
+
 import re
+
 from Products.DataCollector.plugins.CollectorPlugin import CommandPlugin
 from Products.ZenUtils.IpUtil import getHostByName
 
 
+COMMAND_WRAP_TEMPLATE2 = """
```

Comment:
I'm wondering, what does COMMAND_WRAP_TEMPLATE2 stand for? Can it be just COMMAND_WRAP_TEMPLATE?

Reply:
Good catch. I was experimenting with a couple of options, and forgot to rename it when the seconds turned out better. Fixing.